airbyte-cdk/java/airbyte-cdk/dependencies/src/main/kotlin/io/airbyte/commons/exceptions/ConfigErrorException.kt

```diff
@@ -11,6 +11,11 @@ class ConfigErrorException : RuntimeException {
     val internalMessage: String
     val displayMessage: String
 
+    constructor(displayMessage: String) : super(displayMessage) {
```

Comment:
your call, it might be cleaner to just add `@JvmOverloads` to both constructors instead of actually making a new constructor

Reply:
Done. and TIL